MIGRATION.md

```diff
@@ -1,5 +1,32 @@
 # Migration guides
 
+## 3.6.0 to 3.6.x
```

Comment:
not sure what the versions should be here, let me know

Reply:
likely will be 3.6.0 to 3.6.1

Reply:
```
if there are breaking changes (are we removing sendEvent() when we add logConversion()?) then it should probably be 3.7.0
```

Reply:
so 3.6.x - 3.7.x?